auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -497,7 +522,8 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
   showNRErrorModal () {
     this.$refs.addNRDialog.close()
     this.dialogTitle = 'Error Adding Name Request'
-    this.dialogText = ''
+    this.dialogText =
+    'We couldn\'t find a name request associated with the phone number or email address you entered. Please try again.'
```

Comment:
Theoretically, we will never "not find" the name request, because we will be using a search feature and selecting one of the results. But it's possible that the phone or email don't match.

This is a future problem / different ticket. The change here is OK. No more is neede atm.